pkg/plugin/system/system.go

```diff
@@ -47,6 +48,13 @@ type pluginSystem struct {
 	promptRunner  ioutils.PromptRunner
 }
 
+// CustomCommandExecutor requires the Plugin implementations to provide the
+// ExecuteCustomCommand method so that the Core can ask over gRPC for a specific command to
+// be executed. `cmdName` is the name of the custom command the plugin created.
+type CustomCommandExecutor interface {
```

Comment:
This interface is duplicated?

Reply:
Leftover from rebase I think. This one is not being used. Removed